app/CustomFields/product-collection-acf.php

```diff
@@ -17,16 +17,23 @@
         )
       ),
       array (
-        'key' => $key . '_header_image',
-        'label' => 'Product Collection Header Image',
-        'name' => 'product_collection_header_image',
+        'key' => $key . '_background_image',
+        'label' => 'Product Collection Main Background Image',
+        'name' => 'main_background_image',
```

Comment:
the `key` and `name` should be almost identical, apart from in `key` you prefix with `$key . '...'.

example:

```
$key = 'fancy_feature'

array(
  'key' => $key . '_this_is_my_field'
  ...
  'name' => `this_is_my_field`
)
```